### PR TITLE
Add an article about webpack and the spread operator.

### PR DESCRIPTION
**Pull request by Natim**

### CODE REVIEW

File: content/2018.02.using-spread-operator-with-webpack.rst [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+:lang: en
+:date: 2018-02-01
+:url: spread-operator-with-webpack
+:summary: A trick to configure webpack so that it handles the spread operator.
+
+
+What is the spread operator?
+============================
+
+One of the cool feature of JavaScript is the spread operator feature:
```
Comment (leplatrem) [inline content/2018.02.using-spread-operator-with-webpack.rst]: Object rest spread https://github.com/tc39/proposal-object-rest-spread

It's only *stage 3 proposal*. 

Link to https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Spread_operator ?

File: content/2018.02.using-spread-operator-with-webpack.rst [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+
+.. code-block:: javascript
+
+    const preferences = {...defaultPreferences, ...savedPreferences};
+
+Before this feature you could write it like that:
+
+.. code-block:: javascript
+
+    const preferences = _extends({}, defaultPreferences, savedPreferences);
```
Comment (leplatrem) [inline content/2018.02.using-spread-operator-with-webpack.rst]: What's `_extends`? We used to use `Object.assign(....)` !

File: content/2018.02.using-spread-operator-with-webpack.rst [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+    }
+
+
+Conclusion
+==========
+
+This is a simple way to make webpack and the spread operator to work
+nicely together.
+
+Note that this will probably soon be unecessary since webpack is
```
Comment (leplatrem) [inline content/2018.02.using-spread-operator-with-webpack.rst]: unnecessary

File: content/2018.02.using-spread-operator-with-webpack.rst (hunk 1 above)
Comment (Natim) [reply content/2018.02.using-spread-operator-with-webpack.rst]: > It's only stage 3 proposal.

What does it means?

**Review by leplatrem — APPROVED** [review]
I was looking for a CanIUse table, but could not find any...
Mainly because not every browser supports it, it's only since Firefox 55 for example:
https://bugzilla.mozilla.org/show_bug.cgi?id=1339395

That means using Babel is kind of mandatory anyway, even without webpack :)

**Comment by Natim** [comment]
The [MDN page](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Spread_operator) have got a Browser compatibility table.

**Comment by glasserc** [comment]
No strong opinions. I feel like this article is mainly about a language feature which doesn't exist yet, so maybe we should highlight how experimental this feature is (linking to the TC39 proposal), rather than just presenting this as a bug to work around. Of course babel is a tool that you use to fit other experimental features into JS, so to me this whole article is kind of redundant. But I won't fight you if you want to publish redundant things ;)

**Comment by Natim** [comment]
Yes my mistake was to think that because it was implemented in Firefox and Chrome it was a standard. I am not really familiar with JavaScript and I've been using this feature for a while now so to me it was pretty shocking that webpack handled it as a Syntax error.

**Pull request merged**